Note that agent needs to be restarted if the upgrade happened via RPM or DEB (#376)
elastic/ingest-docs: docs/en/ingest-management/commands.asciidoc

IMPORTANT: If you installed {agent} from a DEB or RPM package, the `install`
command will skip the installation itself and function as an alias of the
<<elastic-agent-enroll-command,`enroll` command>> instead. Note that after

> an upgrade of the {agent} via DEB or RPM the {agent} service would need to be restarted.

Reviewer: ```suggestion
an upgrade of the {agent} using DEB or RPM the {agent} service needs to be restarted.
```
